core/node/libp2p/rcmgr_defaults.go

```diff
@@ -108,8 +109,10 @@ func checkImplicitDefaults() {
 	}
 
 	// Check 2: did go-libp2p's SetDefaultServiceLimits change?
-	testLimiter := rcmgr.NewStaticLimiter(rcmgr.DefaultLimits)
-	libp2p.SetDefaultServiceLimits(testLimiter)
+	l := rcmgr.DefaultLimits
+	libp2p.SetDefaultServiceLimits(&l)
+	limits := l.AutoScale()
```

Comment:
Note: `AutoScale` dedicates 1/8 of the system memory to libp2p (which effects how many conns / streams / memory etc.) will be allowed. This might be too little for a standalone go-ipfs node, and too much for IPFS Desktop / Brave.

@lidel Do you have any "profile" setting that we can use here?

Reply:
We do have a [`server` profile](https://github.com/ipfs/kubo/blob/a982068c7a8784d66795b7d73413e6c43fca08f4/config/profile.go#L46-L57) which can adjust this, if we expose it via config as a dedicated toggle.